src/Pelagos/Bundle/AppBundle/Resources/public/js/download.js

```diff
@@ -8,12 +8,14 @@ function startDownload(id)
         return;
     }
 
-    $('<div id="download_splash" />').appendTo("#pelagos-content").load(
```

Comment:
I didn't even have to remove .load, because only the .load event is deprecated, and the load function.